homeassistant/components/sabnzbd/__init__.py

```diff
@@ -31,19 +43,34 @@
 PLATFORMS = ["sensor"]
 _LOGGER = logging.getLogger(__name__)
 
-SPEED_LIMIT_SCHEMA = vol.Schema(
-    {vol.Optional(ATTR_SPEED, default=DEFAULT_SPEED_LIMIT): cv.string}
+SERVICES = (
+    SERVICE_PAUSE,
+    SERVICE_RESUME,
+    SERVICE_SET_SPEED,
+)
+
+SCHEMA_SERVICE = vol.Schema(
+    {
+        vol.Optional(ATTR_SPEED, default=DEFAULT_SPEED_LIMIT): cv.string,
+        vol.Required(ATTR_API_KEY): cv.string,
+    }
 )
 
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_API_KEY): str,
-                vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
-                vol.Required(CONF_URL): str,
-                vol.Optional(CONF_PATH): str,
-            }
+            vol.All(
+                cv.deprecated(CONF_HOST),
+                cv.deprecated(CONF_PORT),
+                cv.deprecated(CONF_SENSORS),
+                cv.deprecated(CONF_SSL),
```

Comment:
Please also restore these options in the dict schema below as they were before the last PR. Otherwise they won't be allowed during config validation.

Reply:
Done